recipes/cpprestsdk/all/conandata.yml

```diff
@@ -1,9 +1,9 @@
 sources:
-  "2.10.15":
-    sha256: 1c027a53457e87b0b3a475e5c8045b94400c475898c8bd51b0fbd218b99a7f7b
-    url: https://github.com/Microsoft/cpprestsdk/archive/v2.10.15.tar.gz
+  "2.10.16":
```

Comment:
please keep old version in place

Reply:
OK... This will need separate versions of the patches per version.